src/nxdoc/web-ui/web-ui-tutorials/web-ui-slots.md

```diff
@@ -266,7 +276,7 @@ This slot is available on a current document that has attached blobs. Default ac
 
 {{#> callout type='note' }}
 - Until Web-UI version `0.9`, this slot was called `BROWSE_ACTIONS` and its properties may have changed.
-{{/callout}}
+  {{/callout}}
```

Comment:
Looks like the spaces are not necessary here and only add noise to the commit, could be removed for a cleaner PR